github/test_download_s3.py

```diff
@@ -0,0 +1,72 @@
+"""
+make sure we can download a file on demand
+"""
+import resources as r
+import pyjq
+import pytest
+
+
+
+@pytest.mark.github_org
+@pytest.mark.rationale(
+        """
+Sanity check for early development
+- can we still retrieve the file
+- can we still parse the file
+- is the data of a size and format we expect
+"""
+)
+@pytest.mark.parametrize(
+        "org,date",
+        [(r.cur_org(), r.report_date()),],
+)
+def test_valid_org_file(org, date):
+    # all the checks to see if we have sane input before proceeding
+    org_data = r.get_data_for_org(org, date)
```

Comment:
Our standard approach has been for the function in parameterize to handle all fetching of data.

Reply:
Let's discuss this -- that doesn't work well for arrays, and this relates to how I've tweaked the handling of metadata.

Also, this test is confirm that the infrastructure has provided the correct file for the run. (The wrapper downloads the org data before a run. If the two get out of sync -- problems.)